test/fixtures/xsd/ProcessMaker.xsd

```diff
@@ -16,6 +16,8 @@
                 <xsd:attribute name="interstitialScreenRef" type="xsd:string"/>
                 <xsd:attribute name="screenVersion" type="xsd:string" use="optional"/>
                 <xsd:attribute name="dueIn" type="xsd:decimal"/>
+                <xsd:attribute name="dueInVariable" type="xsd:string"/>
```

Comment:
Could tried to adding the optional flag like:
`<xsd:attribute name="dueInVariable" type="xsd:string" use="optional"/>`